deployment/helm/values.yaml

```diff
@@ -5,8 +5,11 @@
 # alert - configurations for the Alert Pod
 alert:
   image: "docker.io/blackducksoftware/blackduck-alert:VERSION_TOKEN"
-  limitMemory: "2560M"
-  requestMemory: "2560M"
+  resources:
+    limits:
+      memory: "2560M"
+    requests:
+      memory: "2560M"
```

Comment:
@mphammer this is a bug, similar to the one for Black Duck. https://github.com/blackducksoftware/blackduck-alert/blob/5.2.1/deployment/docker-compose/standalone/docker-compose.yml#L28

M in docker means Mi in k8s

See: https://github.com/blackducksoftware/synopsys-operator/pull/1100/files and https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/

Reply:
Nice find! I switched it to Mi